### PR TITLE
Remove participate tab

### PR DESCRIPTION
**Pull request by jerrycyip**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
Per the latest designs, this PR removes (retires) the "Participate Tab" from the meeting view such that functionality is consolidated in just the meeting agenda view. Code changes correspond to the following [Kanban Trello Card](https://trello.com/c/kkfhOHW3/60-eng-remove-participate-tab-front-end-and-backend-codes)
 
- Describe what changes are being made
References to ParticipateView.jsx and ParticipateView.scss as found in "frontend/src/components/MeetingView/ParticipateView/..." have been retired.  In order to consolidate prior ParticipateView functionality into the remaining Agenda view page, the 'Join Zoom Meeting' button was added to AgendaView.jsx component with corresponding style additions to AgendaView.scss.  Necessary colors, and icon edits were made to support the Join Zoom Meeting button as per the latest [Figma Designs - Community Member](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?node-id=2592%3A59530). Lastly, a default zoom url
link placeholder ("https://zoom.us/join") has been added to the create meeting component to be updated later once a defined mechanism for setting a meeting's zoom url has been designed.  

- Describe why these changes are being made
Per the latest [Figma Designs - Community Member](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?node-id=2592%3A59530) the Participate Tab has been removed with any necessary functionality being consolidated into the Agenda View.  Code changes correspond to the following [Kanban Trello Card](https://trello.com/c/kkfhOHW3/60-eng-remove-participate-tab-front-end-and-backend-codes)

- List the use cases and edge cases relevant to this PR
Use Case: Community user viewing the meeting and meeting agenda.  Edge Cases: N/A.  Code changes correspond to the following [Kanban Trello Card](https://trello.com/c/kkfhOHW3/60-eng-remove-participate-tab-front-end-and-backend-codes)

- Describe how errors will be handled. How will we know if this code breaks in production
N/A

- Describe any external libraries/dependencies added or removed in this PR
N/A

- Describe any security risks are there regarding this change
N/A

- Describe how you tested these changes
Navigated through user flow for viewing a meeting as both an admin and community user to confirm corresponds with designs.

- Link to relevant external documentation
See latest [Figma Designs - Community Member](https://www.figma.com/file/MEBv8hZLr4MICzKCVXADjg/Happening-ATM?node-id=2592%3A59530) the Participate Tab has been removed with any necessary functionality being consolidated into the Agenda View.

--------------------------
### :clipboard: Mandatory Checklist

- [x] Example of a checked item (please remove when creating your Pull Request) 

- [x] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [x] Have you discussed these changes with the project leader(s)?
- [x] Do all variable and function names communicate what they do?
- [x] Were all the changes commented and / or documented?
- [x] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [x] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [x] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### Before

https://user-images.githubusercontent.com/54079796/150461223-cab35998-85a6-4c96-b73e-6e520d8fc9bf.mov

#### After

https://user-images.githubusercontent.com/54079796/150460804-5a7b5dbc-71e9-4e56-a764-a58318ce3aa5.mov

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

File: frontend/src/components/MeetingView/MeetingView.jsx [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
+import { GET_MEETING_WITH_ITEMS } from "../../graphql/graphql";
 
+import NavBarHeader from "../NavBarHeader/NavBarHeader";
+import Header from "../Header/Header";
+//import ParticipateView from "./ParticipateView/ParticipateView";
+import AgendaView from "./AgendaView/AgendaView";
+import Spinner from "../Spinner/Spinner";
+import {
+  JoinMeetingIcon,
+} from '../../utils/_icons';
```
Comment (ppelayo1) [inline frontend/src/components/MeetingView/MeetingView.jsx:16]: Your ESlint seems to be having issues on the auto correct end. Probably a global issue as I ran your code, and looking at it thru my editor and it formatting correctly.

File: frontend/src/components/MeetingView/MeetingView.jsx [hunk 2]
```diff
@@ -91,43 +93,35 @@ function MeetingView() {
...
+          <button 
+          type="button"
+          className="join-meeting"
           >
-            {t('meeting.tabs.participate.label')}
+          <JoinMeetingIcon/>          
+            <p>{t(
+              "meeting.tabs.participate.section.join.description.number-2.button"
+            )}</p>
           </button>
```
Comment (ppelayo1) [inline frontend/src/components/MeetingView/MeetingView.jsx:113]: Your ESlint having issue here too. It formats correctly on my end in my Visual Studio Code.

**Comment by ppelayo1** [comment]
I left some comments, looks like your ESlint having issues. Probably something on the global end.

File: frontend/src/components/MeetingView/MeetingView.jsx [hunk 3]
```diff
@@ -37,22 +37,24 @@ function MeetingView() {
   // queries
   const { loading, error, data } = useQuery(GET_MEETING_WITH_ITEMS, {
     variables: { id: parseInt(id, 10) },
-    fetchPolicy: 'network-only',
+    fetchPolicy: "network-only",
   });
 
   // states
   const [meetingWithItems, setMeetingWithItems] = useState({});
-  const [showAgendaView, setShowAgendaView] = useState(true);
```
Comment (ppelayo1) [inline frontend/src/components/MeetingView/MeetingView.jsx:45]: Removing this breaks the ability for the admin to save in the Header.jsx.

If you go into Header.jsx and remove it as an argument, and then remove the condition requirement on line 72 to check only for isAdmin() we can continue to save changes in item order until we get a new design implemented.

**Comment by ppelayo1** [comment]
Looks good, just a few small things I noticed.

**Comment by jerrycyip** [comment]
updated code to reflect feedback from Patrick's review (e.g. fixes for linter formatting and ability of admin to update status in header).  In addition resolved 3 (hopefully minor) merge conflicts related to recent PRs that updated main 'Develop' branch to support showing the meeting status -- for the most part adopted code as existing on main Develop branch.

**Review by ppelayo1 — APPROVED** [review]
Looks fine, your lint still having a problem with those quotes though.

**Pull request merged**